solidity_deposit_contract/README.md

```diff
@@ -0,0 +1,25 @@
+# eth2-deposit-contract
```

Comment:
/cc @axic 
I added a note of `r2`. Let me know if you want to add something. :)